src/Types.hs

```diff
@@ -107,6 +107,9 @@ app2 f g x y = f $ g x y
 sortOn :: Ord b => Getter a b -> [a] -> [a]
 sortOn l = sortBy (compare `on` view l)
 
+sortOnDesc :: Ord b => Getter a b -> [a] -> [a]
+sortOnDesc l = reverse . sortOn l
```

Comment:
Can you rename Desc to Down as this is the convention in Data.Ord?

And then this is: `downSortOn l = sortOn (l . to Down)`